tests/rados/test_scrub_parameters.py

```diff
@@ -0,0 +1,204 @@
+            if not forced_flag:
+                log.error(
+                    "The forced flag is not set to True and the waiting time is 20 minutes"
+                )
+                return 1
+            log.info(f"===Scenario-1: Completed by {val}bing the osd===")
+            pg_dump_scrub = scrub_object.get_pg_dump_scrub(selected_osd, pg_id)
+            pg_current_sched_time = pg_dump_scrub["sched_time"]
+            # Get the dump_scrub of the PG id
+            scheduled_time = verify_scheduled_time(
+                mon_object=mon_obj,
+                prev_dump_scrub=dump_scrub_before_scrub,
+                pg_id=pg_id,
+                pg_current_sched_time=pg_current_sched_time,
+            )
+            if not scheduled_time:
+                log.error("The scrub time is greater than upper bound time")
+                return 1
+            log.info(f"=====Verification completed by {val}bing the OSD====")
+            # deleted pool.This is due to execution time is taking more time until waiting for the cluster to clean
+            # state
+            method_should_succeed(rados_object.delete_pool, pool_name)
+            rados_object.log_cluster_health()
+            log.info(f"===Scenario-2:Completed by {val}bing the osd")
+    except Exception as e:
+        log.info(e)
+        log.info(traceback.format_exc())
+        return 1
+    finally:
+        log.info("Execution of finally block")
+        rados_object.log_cluster_health()
+    return 0
+
+
+def create_pool_get_osd(rados_object, pool_name):
+    """
+    Method is used -
+        1. Create replicated pool
+        2. Push the data into pool
+        3. From the active OSD list return a random OSD
+    Args:
+        rados_object: Rados object
+        pool_name: pool name
+    Returns:
+        Return a random OSD ID from an active OSD list
+    """
+    if not rados_object.create_pool(pool_name=pool_name):
+        log.error("Failed to create the replicated Pool")
+        return 1
+    # rados_object.bench_write(pool_name=pool_name)
+    osd_list = rados_object.get_active_osd_list()
+    osd_id = random.choice(osd_list)
+    return osd_id
+
+
+def chk_forced_flag(scrub_object, osd_id):
+    """
+    Method is used to check the forced flag status
+    Args:
+        rados_object: Rados object
+        osd_id: OSD ID number
+    Returns:
+        True -> If forced flag is true
+        False -> If forced flag is false
+    """
+    pg_id = ""
+    time_execution = datetime.datetime.now() + datetime.timedelta(minutes=20)
+    found = False
+    while datetime.datetime.now() < time_execution:
+        out_put = scrub_object.get_dump_scrubs(osd_id)
+        for pg_no in out_put:
+            if pg_no["forced"] is True:
+                pg_id = pg_no["pgid"]
+                log.info(f"The pg with the pg-id {pg_id} is true")
+                found = True
+                break
+        if found:
```

Comment:
If found, should we not return the values? ` return True, pg_id ` Instead of break?

Reply:
Done